src/js/common/components/RequiredTermsAcceptanceView.jsx

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { checkAcceptance } from '../../user-profile/actions';
+
+import AcceptTermsPrompt from './AcceptTermsPrompt';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+
+class RequiredTermsAcceptanceView extends React.Component {
+  componentWillMount() {
+    // I get errors and the app crashes if this line is called.
+    // this.props.checkAcceptance();
```

Comment:
@aub what is the error that you see? This line kicks off the check which should ping the API for the acceptance check. It definitely should not crash so if you have more context I can see what's going on. 

My app doesn't crash but I get a 404 for the check which I presume means that the user has not accepted the terms.

```
code: "404"
detail: "The record identified by mhvac could not be found"
```